src/cascadia/TerminalApp/App.cpp

```diff
@@ -1036,7 +1036,10 @@ namespace winrt::TerminalApp::implementation
             {
                 try
                 {
-                    return _GetFocusedControl().Title();
+                    if (auto focusedControl{ _GetFocusedControl() })

```

Comment:
There's a chance there's a similar bug with scrolling using the keyboard, if you want to fold that into this PR

Reply:
I think there are some issues in tying "active control" to "control currently focused in the XAML stack".. what do you think? Should we just holistically fix this all up by making active/focused not the same?

Reply:
oh that's a great idea

Reply:
Moved to #1205.